integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/CatalogsPage.java

```diff
@@ -403,6 +403,26 @@ public boolean verifyShowTableTitle(String title) {
     }
   }
 
+  public boolean verifyShowDataItemInList(String itemName) {
+    try {
+      List<WebElement> list =
+          tableGrid.findElements(
+              By.xpath(
+                  "./div[contains(@class, 'MuiDataGrid-main')]/div[contains(@class, 'MuiDataGrid-virtualScroller')]/div/div[@role='rowgroup']"));
```

Comment:
better change `rowgroup` to `rowGroup`?

Reply:
This named string was generated by the tree component.